ceph.spec.in

```diff
@@ -257,7 +250,17 @@ BuildRequires:	gcc-c++ >= 11
 BuildRequires: gcc11-c++
 %endif
 %if 0%{?gts_version} > 0
+%if 0%{?gts_version} == 13
+# Before 13.3, an lto bug resulted in a segfault in SafeTimer and perhaps
+# elsewhere. Require the fixed version so we can reenable lto. See
+# ceph bug https://tracker.ceph.com/issues/63867
+# and
+# gcc bug https://bugzilla.redhat.com/show_bug.cgi?id=2241339
+# for details.
+BuildRequires:	gcc-toolset-%{gts_version}-gcc-c++ >= 13.3
```

Comment:
Can we require 13.3 and above regardless? Meaning bumping the requirement from 11 to 13.3.